src/lib/mongoBackend/mongoUpdateSubscription.cpp

```diff
@@ -457,6 +458,55 @@ static void setLastNotification(const BSONObj& subOrig, CachedSubscription* subC
 }
 
 
+
+/* ****************************************************************************
+*
+* setLastFailure -
+*/
+static long long setLastFailure(const BSONObj& subOrig, CachedSubscription* subCacheP, BSONObjBuilder* b)
```

Comment:
I understand that the setLastFailure() and setTimesFailed() method are here because at the end we chose to keep lastFailure and timesFailed during a subscription update (vs. assuming that the subscription has changed too much and they should be resealed), isn't it? (I don't remember)